src/redisearch_rs/tools/shadow_tester/src/args.rs

```diff
@@ -0,0 +1,20 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+
+#[derive(Parser)]
+#[command(version, about)]
+pub struct Options {
+    /// Path to the baseline .so file. The responses from this module will be considered the source
+    /// of truth.
+    #[arg(short, long)]
+    pub baseline_so: PathBuf,
+
+    /// Path to the changeset .so file. The responses from this module should match the baseline's.
+    #[arg(short, long)]
+    pub changeset_so: PathBuf,
+
+    /// Command to start up RLTest which will start a Redis server for each .so file.
+    #[arg(short, long)]
+    pub rltest_command: String,
```

Comment:
Isn't this a path too?

Reply:
Yes, in our usage but technically it can be a command too. Eg I think it will work when given `python3 -m RLTest ...` too. So I'm torn between making it a path or a string.

Reply:
Giving it a command won't work. I'm changing this to a path too.